d2l-colors.html

```diff
@@ -44,6 +44,12 @@
 				--d2l-color-white: #fff;
 				--d2l-color-woolonardo: #f2f3f5;
 
+				/* alerts */
```

Comment:
Cool.  I am guessing that these are for more than alerts (this they are going into d2l-colors)?